setup.py

```diff
@@ -51,10 +51,10 @@
 }
 
 extras_require['dev'] = (
-    extras_require['tester'] +
-    extras_require['linter'] +
-    extras_require['docs'] +
-    extras_require['dev']
+    extras_require['tester']
+    + extras_require['linter']  # noqa: W503
```

Comment:
I would be ok with adding `W503` to the ignored warnings in our `tox.ini` file.

Reply:
yeah. right! will do